Copy to SD Card root directory to update/config.ini

```diff
@@ -1,282 +1,413 @@
+# Format: [move_speed: S<feedrate mm/min> N<feedrate mm/min> F<feedrate mm/min>]
+#
+# Unit: [mm/min]
+#
+# Options: [slow: S, normal: N, fast: F]
+#
 move_speed:S1000 N3000 F5000
 
-#### default Extruder Speeds (mm/min)
-# Options: [Slow: S, Normal: N, Fast: F]
-# format   [ext_speed: S<feedrate mm/min> N<feedrate mm/min> F<feedrate mm/min>]
+#### Default Extruder Speeds
+# Format: [ext_speed: S<feedrate mm/min> N<feedrate mm/min> F<feedrate mm/min>]
+#
+# Unit: [mm/min]
+#
+# Options: [slow: S, normal: N, fast: F]
+#
 ext_speed:S60 N600 F1200
 
 #### Auto Save Load Leveling Data
 # The TFT will auto-detect if Auto Bed Level is available.
 # Enable this will send "M500" after "G29" to store leveling value
 # and send "M420 S1" to enable leveling state after startup
+#
 # Options: [enable: 1, disable: 0]
+#
 auto_load_leveling:1
 
 #### Onboard / Printer SD Card Support
-# On Marlin Firmware, The TFT will auto detect Onboard SD Card
+# On Marlin firmware, the TFT will auto-detect Onboard SD Card.
 # Auto-detect is not available for other firmwares like Smoothieware
+#
 # Options: [enable: 1, disable: 0, auto-detect: 2]
+#
 onboard_sd_support:2
 
-#### M27 / Printing status refresh time (this will be used if SD_AUTOREPORT is not detected by the TFT)
+#### M27 Printing Status Refresh Time
+# M27 printing status refresh time (this will be used if SD_AUTOREPORT is not detected by the TFT)
+#
+# Unit: [seconds]
+#
 M27_refresh_time:3
 
-#### M27 always active - keep polling M27 even if not printing
+#### M27 Always Active
+# Keep polling M27 even if not printing
+#
 # Options: [enable: 1, disable: 0]
+#
 M27_always_active:1
 
 #### Long File Names Support
-# On Marlin Firmware, The TFT will auto-detect Long File Name supported
+# On Marlin firmware, the TFT will auto-detect Long File Name support.
 # Auto-detect is not available for other firmwares like Smoothieware
+#
 # Options: [enable: 1, disable: 0, auto-detect: 2]
+#
 long_filename_support:2
 
-#### Show Fan Speed as Percentage
+#### Fan Speed As Percentage
+# Show fan speed as percentage
+#
 # Options: [enable: 1, disable: 0]
+#
 fan_speed_percent:1
 
 #### Pause Settings
-# position & Lengths in mm, Feedrates in mm/min
-# minimum possible value: 0, maximum possible value: 2000
-# Pause retract distance format [pause_retract: R<retract length> P<Resume Purge length>]
-# Pause XY position format      [pause_pos: X<position in mm> Y<position in mm>]
-# Pause Z raise format          [pause_z_raise: Z<distance in mm>]
-# Pause feed Rate # format      [ X<feedrate mm/min> Y<feedrate mm/min> Z<feedrate mm/min> E<feedrate mm/min>]
+# Format: [pause_retract: R<retract length> P<Resume Purge length>]
+#         [pause_pos: X<position in mm> Y<position in mm>]
+#         [pause_z_raise: Z<distance in mm>]
+#         [pause_feedrate: X<feedrate mm/min> Y<feedrate mm/min> Z<feedrate mm/min> E<feedrate mm/min>]
+#
+# Unit: [position & length: mm, feedrate: mm/min]
+#
+# Value range: [min: 0, max: 2000]
+#
```

Comment:
Do not remove the description/name of the config lines.

Reply:
not my intention. Moved param names at the beginning of the section for Pause and Leveling settings

Reply:
instead of a separate list for names and a separate list of formats, it would be better if the param names are clubbed together with their respective format.

Reply:
well, those must be  param description not param names. Param names are already reported in the format. Also reporting them in the same line will make it less readable. You now always easily identify tags such as Format, Unit etc, while they were previously  mixed with param name/description etc... For the 2 cases you refer to, it should be better to provide a description instead of a fully qualified param name (for that reason I previosly removed them. They didn't add any description)